packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -393,6 +393,8 @@ class AppDomain extends Domain {
 
     if (options.debuggingEnabled) {
       connectionInfoCompleter = new Completer<DebugConnectionInfo>();
+      // Callback hookups' Futures are uninteresting.
+      // ignore: unawaited_futures
       connectionInfoCompleter.future.then<Null>((DebugConnectionInfo info) {
```

Comment:
In this case, we're more curious about what happens if connectionInfoCompleter's 'then' callback fails rather than the app itself. If the callback fails to _sendAppEvent somehow after the app successfully starts, what should we do?